crates/ark/src/lsp/completions/sources/unique/custom.rs

```diff
@@ -106,6 +106,13 @@ pub fn completions_from_custom_source_impl(
         tower_lsp::lsp_types::ParameterLabel::Simple(string) => string,
     };
 
+    // Argument text typically contains the argument name and its default value if there is one.
+    // Extract out just the argument name for matching purposes.
+    let argument = match argument.find("=") {
```

Comment:
I'd rename `let argument` to `let formal` or `let param` since "argument" normally refers to the value passed in.

Reply:
oh I see above we use "parameter" which is clearer. Why did we go from param to arg terminology? Since we're analysing a function rather than a function call, there is no argument involved except for the default arg if there is one. I think it helps to be accurate and consistent in this sort of code.